src/Components/Inputs/Input.tsx

```diff
@@ -79,6 +80,13 @@ export interface InputProps extends StandardFunctionProps {
   inputStyle?: CSSProperties
   /** For use within a form, marks the input as required */
   required?: boolean
+  /**
+   * Function to be called when the input field is cleared;
+   * if not included then the 'x' clear button will NOT be added
+   * */
+  onClear?: () => void
+  /** optional extra styling for the clear button */
+  clearBtnStyle?: CSSProperties
```

Comment:
Bit of a redundant feedback here but Not sure if we need to expose this. Im trying to think of a case where we would need a special css treatment of the dismiss button and can't think of it.

Reply:
i needed it previously, then after adjusting for component size don't need it; but it was already there so I left it in.

Reply:
...just removed it